topicchanger.go

```diff
@@ -129,7 +132,8 @@ SELECT
   override,
   CASE WHEN location = '' OR location IS NULL THEN 'TBA' ELSE location END,
   termine.date,
-  CASE WHEN topic = '' OR topic IS NULL THEN 'noch keine ◉︵◉' ELSE topic END
+  CASE WHEN topic = '' OR topic IS NULL THEN 'noch keine ◉︵◉' ELSE topic END,
```

Comment:
sollte das nicht  `ELSE topic, speaker END` heißen?

Reply:
Nein, dass `END` bezieht sich ja auf `CASE` und dann müsste es ggf.
`CASE WHEN topic = '' OR topic IS NULL THEN 'noch keine ◉︵◉', '' ELSE topic, speaker END,`
heißen.
Man muss auf jeden Fall eine zusätzliche Spalte im Query returnen, siehe https://github.com/nnev/frank/pull/53/files#diff-40a80c6797b984151295c9c0ab35a03aL153